src/rules/block-no-empty/index.js

```diff
@@ -27,7 +27,7 @@ export default function (actual) {
       let index = beforeBlockString(statement, { noRawBefore: true }).length
 
       // For empty blocks when using SugarSS parser
-      if (statement.raw("between") === undefined) {
+      if (statement.raws.between === undefined) {
```

Comment:
Sorry I missed it above. I don't like the checklist in the template, makes me skip reading.

SGTM. Also found `raws` in the PostCSS docs which for some reason I didn't find before, so I'm more comfortable using it.